### PR TITLE
feat(github): added stale bot configuration to clean up obsolete PRs and Issues

### PR DESCRIPTION
**Pull request by chefsale**

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+issues:
+  # Number of days of inactivity before an issue or PR becomes stale
+  daysUntilStale: 90
```
Comment (bowenwang1996) [inline .github/stale.yml:3]: Could we do 60 instead?

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+  # Number of days of inactivity before a stale issue or PR is closed
+  daysUntilClose: 7
+
+  # Issues or PRs with these labels will never be considered stale
+  exemptLabels:
+    - S-pinned
+    - security
```
Comment (bowenwang1996) [inline .github/stale.yml:11]:
```suggestion
    - A-security
```

File: .github/stale.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
+issues:
+  # Number of days of inactivity before an issue or PR becomes stale
+  daysUntilStale: 90
+
+  # Number of days of inactivity before a stale issue or PR is closed
+  daysUntilClose: 7
+
+  # Issues or PRs with these labels will never be considered stale
+  exemptLabels:
+    - S-pinned
```
Comment (bowenwang1996) [inline .github/stale.yml:10]: Do we have this label?

File: .github/stale.yml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  # Number of days of inactivity before an issue or PR becomes stale
+  daysUntilStale: 14
+
+  # Number of days of inactivity before a stale issue or PR is closed
+  daysUntilClose: 3
+
+  # Issues or PRs with these labels will never be considered stale
+  exemptLabels:
+    - S-pinned
+    - security
```
Comment (bowenwang1996) [inline .github/stale.yml:41]: Same here

File: .github/stale.yml (hunk 3 above)
Comment (chefsale) [reply .github/stale.yml:10]: No, but we jist need to add it.

File: .github/stale.yml (hunk 4 above)
Comment (chefsale) [reply .github/stale.yml:41]: Same as above.

File: .github/stale.yml (hunk 1 above)
Comment (chefsale) [reply .github/stale.yml:3]: Sure, I will change that.

**Comment by chefsale** [comment]
Changes applied. cc: @bowenwang1996

**Review by mhalambek — APPROVED** [review]

**Pull request merged**